### PR TITLE
Feature navigation

### PR DESCRIPTION
**Pull request by Bhakti199**
here I added a standard navigation.

### CODE REVIEW

**Review by paarthbajaj — APPROVED** [review]
Well-written code. You can merge.

File: navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
+@import url(/style.css);
+@import url(/badge/badge.css);
+
```
Comment (rkapoor10) [inline navigation/navigation.css:3]: most of your code was in the initial commit. Try to change the commit message before pushing origin.

File: navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+.nav-intro-heading,
+.nav-intro {
+  text-align: center;
+}
+.nav-container {
+  min-width: 40rem;
+  height: auto;
+  margin: 3rem 2rem 2rem 2rem;
+}
+.nav-standard-item {
```
Comment (rkapoor10) [inline navigation/navigation.css:13]: this property is just adding flex you may have a class - "flex-row" or "flex-column" in utility CSS and use that,

**Review by rkapoor10 — APPROVED** [review]
Code looks good; approved for merge

File: navigation/navigation.css (hunk 2 above)
Comment (Bhakti199) [reply navigation/navigation.css:13]: yes, I will

**Pull request merged**